package com.rakovets.course.java.core.practice.exception_handling;

public class Task2 {
        public void exceptionArray() {
            int[] array = {3, 4, 5};
            array[3] = 6;
        }
    }
